src/hearings/containers/hearing-actuals/hearing-stage-result/hearing-stage-result.component.html

```diff
@@ -1,82 +1,93 @@
-                  <label class="govuk-label govuk-radios__label" for="adjourned">
-                    Part-heard, adjourned or postponed
-                  </label>
-                </div>
-                <div class="govuk-radios__conditional" *ngIf="hearingResultType === hearingResultEnum.ADJOURNED">
-                  <div class="govuk-form-group">
-                    <label class="govuk-label" id="dropdown-label" for="adjourned-reason">Select a reason for this result</label>
-                    <select formControlName="adjournedReason" class="govuk-select" id="adjourned-reason" name="adjournedReason">
-                      <option value="" selected>Please select</option>
-                      <option *ngFor="let reason of adjournHearingActualReasons" [ngValue]="reason.key">{{reason.value_en}}</option>
-                    </select>
+        <div class="govuk-form-group"
+          [ngClass]="{'govuk-form-group--error': submitted && !f.valid && !f.hearingResult.valid}">
+          <fieldset class="govuk-fieldset">
+            <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
+              <h1 class="govuk-fieldset__heading">
+                Hearing result
+              </h1>
+              <div class="govuk-error-message" *ngIf="submitted && !f.valid && !f.hearingResult.valid">
+                {{hearingStageResultEnum.HearingResultError}}
+              </div>
+            </legend>
+            <div class="govuk-radios" data-module="govuk-radios">
+              <div class="govuk-radios__item">
+                <input class="govuk-radios__input" id="completed" formName="hearingResult" name="hearingResult"
+                  type="radio" value="completed" data-aria-controls="conditional-completed" (change)="onHearingResult(hearingResultEnum.COMPLETED)">
+                <label class="govuk-label govuk-radios__label" for="completed">
+                  Heard, order or decision
+                </label>
+              </div>
+              <div class="govuk-radios__item">
+                <input class="govuk-radios__input" id="adjourned" formName="hearingResult" name="hearingResult"
+                  type="radio" value="adjourned" data-aria-controls="conditional-adjourned" (change)="onHearingResult(hearingResultEnum.ADJOURNED)">
+                <label class="govuk-label govuk-radios__label" for="adjourned">
+                  Part-heard, adjourned or postponed
+                </label>
+              </div>
+              <div class="govuk-radios__conditional" *ngIf="hearingResultType === hearingResultEnum.ADJOURNED">
+                <div class="govuk-form-group"
+                  [ngClass]="{'govuk-form-group--error': submitted && adjournHearingErrorMessage.length > 0}">
+                  <label class="govuk-label" id="dropdown-label" for="adjourned-reason">Select a reason for this result</label>
```

Comment:
duplicated 'dropdown-label'

Reply:
Thank you for finding this, I have now changed from id to class.